public/assets/styles/variables.scss

```diff
@@ -0,0 +1,20 @@
+// color variables
+$nav-red: #ff0000;
+$nav-item-black: #3a3a3a;
+$color-accent: #007fff;
+$color-bg: #fff;
```

Comment:
Always write out the colours in full. `#ffffff` not `#fff`

Reply:
feedback well recieved